lib/bandit/http1/handler.ex

```diff
@@ -109,9 +111,17 @@ defmodule Bandit.HTTP1.Handler do
   defp code_for_reason(_), do: 400
 
   defp attempt_to_send_fallback(req, code) do
-    Bandit.HTTP1.Adapter.send_resp(req, code, [], <<>>)
-  rescue
-    _ -> :ok
+    receive do
+      @already_sent ->
+        send(self(), @already_sent)
+    after
+      0 ->
+        try do
+          Bandit.HTTP1.Adapter.send_resp(req, code, [], <<>>)
+        rescue
+          _ -> :ok
+        end
+    end
```

Comment:
it's such an obvious solution but I never would have though of this 🤔 💡 👍

Reply:
Cribbed directly from plug/error_handler.ex source: https://github.com/elixir-plug/plug/blob/v1.15.2/lib/plug/error_handler.ex#L103